pragma-node/src/handlers/entries/mod.rs

```diff
@@ -134,7 +137,11 @@ pub struct GetOnchainCheckpointsResponse(pub Vec<Checkpoint>);
 
 #[derive(Debug, Deserialize, IntoParams, ToSchema)]
 pub struct GetEntryParams {
-    pub timestamp: Option<u64>,
+    /// The unix timestamp in seconds. This endpoint will return the first update whose
+    /// timestamp is <= the provided value.
+    #[param(value_type = i64)]
+    #[param(example = doc_examples::timestamp_example)]
```

Comment:
what is `param(example` used for?

Reply:
to display an example value in the swagger ui, here to make it clear that the timestamp is in seconds, could be added for other fields too